sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRowConverter.scala

```diff
@@ -188,13 +188,13 @@ private[parquet] class ParquetRowConverter(
    */
   def currentRecord: InternalRow = currentRow
 
-  private val dateRebaseFunc = DataSourceUtils.creteDateRebaseFuncInRead(
+  private val dateRebaseFunc = DataSourceUtils.createDateRebaseFuncInRead(
```

Comment:
Oh which of the PRs has the method rename? I wonder if others will fail if they don't have that change as well.

Reply:
Sigh, yeah, this'll definitely fail. I've hit this repeatedly while dropping things. Lemme drop it from here.

It can merge in the root PR.